sdks/java/core/src/test/java/org/apache/beam/sdk/io/DefaultFilenamePolicyTest.java

```diff
@@ -48,6 +51,26 @@ private static String constructName(
     return constructed.toString();
   }
 
+  private static DefaultFilenamePolicy.Params encodeDecodeParams(
```

Comment:
Use [CoderUtils.clone](https://github.com/apache/beam/blob/f3ac4822191ca63d74a8fd71f81da976b3d2fbd1/sdks/java/core/src/main/java/org/apache/beam/sdk/util/CoderUtils.java#L140)

Reply:
thanks for the advice :+1: